.linguirc.json

```diff
@@ -11,9 +11,5 @@
     "origins": false
   },
   "orderBy": "messageId",
-  "fallbackLocales": {
-    "default": "en"
-  },
-  "locales": ["en", "zh", "ru", "tr", "es", "pt", "fr"],
-  "sourceLocale": "en"
+  "locales": ["zh", "ru", "tr", "es", "pt", "fr"]
```

Comment:
Remove english as a lingui lang. The source file is now the `.pot`.

If a translation for a string doesn't exist for a given lang, lingui falls back to the message id.(which is the english string)